internal/provisioner/nginx_internal.go

```diff
@@ -89,6 +93,25 @@ func (n *nginxInternal) ActualVersion() *model.HelmUtilityVersion {
 }
 
 func (n *nginxInternal) Destroy() error {
+	k8sClient, err := k8s.NewFromFile(n.kubeconfigPath, n.logger)
+	if err != nil {
+		return errors.Wrap(err, "failed to set up the k8s client")
+	}
+
+	services, err := k8sClient.Clientset.CoreV1().Services("nginx-internal").List(context.TODO(), metav1.ListOptions{})
```

Comment:
I think `"nginx-internal"` is stored somewhere as a `const`. Would be nice to use that on every instance now that you're on it.

Reply:
Right. We do not have any `const` now. But I will set one.